pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -459,18 +477,42 @@ func (m *kubeGenericRuntimeManager) getPodContainerStatuses(uid kubetypes.UID, n
 func toKubeContainerStatus(status *runtimeapi.ContainerStatus, runtimeName string) *kubecontainer.ContainerStatus {
 	annotatedInfo := getContainerInfoFromAnnotations(status.Annotations)
 	labeledInfo := getContainerInfoFromLabels(status.Labels)
+	var rLimits v1.ResourceList
+	if utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScaling) {
+		var cpuLimit, memLimit *resource.Quantity
+		if status.Resources.CpuShares > 0 {
+			//BUGBUG: TODO: Correctly compute cpuLimit from CpuShares, CpuQuota, CpuPeriod
```

Comment:
is this still a bug?

Reply:
i don't think so, but i need to add thorough unit tests around it to verify all cases are covered. i have it there to get my attention when i come back to this after implementing resize policy.

Reply:
please remove if it's done with.

Reply:
+1. we should remove it.

Reply:
i found a problem when adding unit tests yesterday. i'm replacing this.

Reply:
fixed in the latest commit.